compiler/rustc_borrowck/src/region_infer/opaque_types/mod.rs

```diff
@@ -66,7 +66,7 @@ pub(crate) enum DeferredOpaqueTypeError<'tcx> {
 pub(crate) fn clone_and_resolve_opaque_types<'tcx>(
     infcx: &BorrowckInferCtxt<'tcx>,
     universal_region_relations: &Frozen<UniversalRegionRelations<'tcx>>,
-    constraints: &mut MirTypeckRegionConstraints<'tcx>,
+    placeholder_to_region: &mut PlaceholderToRegion<'tcx>,
```

Comment:
I feel like it might be nicer to keep `PlaceholderToRegion` as a field of `MirTypeckRegionConstraints`? We effectively need this map anytime we do stuff with these constraints, do we not